src/item_factory.h

```diff
@@ -3,6 +3,7 @@
 
 #include "json.h"
 #include "iuse.h"
+#include "bodypart.h"
```

Comment:
`bp_num` is later referenced. Before, it was included as a part of some other header, now it's there explicitly.